realtime/redisPublisher.js

```diff
@@ -137,7 +137,7 @@ function publishSample(sampleInst, subjectModel, event, aspectModel) {
       sample.absolutePath = sample.subject.absolutePath; // reqd for filtering
       publishObject(sample, eventType);
       return sample;
-    }
+    } // What happend here if is invalid??
```

Comment:
@iamigo just pushed this code for double check... redisPublisher call attachAspect that now can throw an exception which is ok here... the .catch might be handled from BulkUpsert... anyway, IF for some reason this sample is null/undefined?